libs/sdk-ui-tests/.storybook/main.js

```diff
@@ -9,4 +11,18 @@ module.exports = {
     core: {
         builder: "webpack5",
     },
+    webpackFinal: async (config) => ({
```

Comment:
Copying the whole build output of the `sdk-ui-web-components` because it's using chunks in the output and Webpack does not understand chunks when used as input.